pkg/lib/page.scss

```diff
@@ -160,10 +160,7 @@ a.disabled:hover {
   --ct-color-list-critical-alert-text: var(--pf-v6-global--palette--red-8888);
 }
 
-// Let PF4 handle the scrolling through page component otherwise we might get double scrollbar
 html:not(.index-page) body {
-  overflow-block: hidden;
```

Comment:
I don't see the `overflow-y: auto` anymore... Did you get the same effect by removing this line?

Reply:
yes

Reply:
but we will likely end up doing something completely different. I'm working with @garrett suggestions from this gist: https://gist.github.com/garrett/1b7dc73b79c97a2b24a5dd02d499e8f4